### DISCUSSION TITLE
How  and are git extension supported in stackblitz? (Where the source? How does it work?)

### DISCUSSION THREAD

**Discussion by frog-o** [stackblitz/core#2575] [Q&A]
In one of your videos i watched you talked about how you created a special version of git for use on stackblitz(i did not Check the date) i am currently looking at the doc from 

https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#getting-started

Wich said it uses git-lfs(git large file system extension). I was wondering what it would take to git this going in stackblitz or if it was already done you mentioned how you ported a lot of the common utilities already for use with stackblitz but didn't say much about how they work or Wich one.

I don't know much about it but i currently looking at how the git-lfs works wich lead me to look into git hooks.I imagine a lot of this depends on the extension. Most are scripts if i am not mistaken.   You say in your docs that python is not supported but i can't help but wonder with the event that python now work in the browser with help from wasm ,(pyscript has a lot of info on this)if this could be done.

I kind of surprised that since most of the main uses for stackblitz is for use with GitHub it doesn't even hardly mention it( unless I missing it)

**Comment by kc0tlh** [comment]
Checkout the extensions tab on [our new VS Code editor](https://stackblitz.com/codeflow)!
